rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
Ok, cool, so this is what I want to be able to do:

```javascript
// when using ESM
import glamorous, {ThemeProvider} from 'glamorous'

// when using CJS
const glamorous = require('glamorous')
const {ThemeProvider} = glamorous

// when using globals
const glamorous = window.glamorous
const {ThemeProvider} = glamorous
```

I'm pretty sure this can be accomplished if we do:

```javascript
Object.assign(glamorous, {ThemeProvider})

export default glamorous
export {ThemeProvider}
```

It's kinda monkey-patching, but hey, this is JavaScript :smile: